canonical/ubuntu.com
# /download/desktop refresh

## Done

- Applied redesign as per [doc](https://docs.google.com/document/d/1ScuTtdrm3-iAvizTDxksVTAerV6QkCUN8Sg60uUlJsk/edit) and [mock-up](https://www.figma.com/file/cVGimTPHky4h3BPXfrT6GB/24.04-U.com---download-(rebranding)?node-id=1%3A30&mode=dev)
- Drive bys:
  - Renamed some existing partials for increased specificity (eg: /_resources -> /_instructions-resources). 
  - Divided the corresponding thank-you page into partials which this landing page also reuses

## QA

- View the site locally in your web browser at: https://ubuntu-com-13697.demos.haus/download/desktop
    - Be sure to test on mobile, tablet and desktop screen sizes
- Check against doc and design

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-8491
